packages/salesforcedx-vscode-apex-replay-debugger/src/index.ts

```diff
@@ -254,9 +257,7 @@ function getDialogStartingPath(): vscode.Uri | undefined {
       return vscode.Uri.file(sfdxCommandLogDir);
     }
     // If all else fails, fallback to the .sfdx directory in the workspace
-    return vscode.Uri.file(
-      path.join(vscode.workspace.workspaceFolders[0].uri.fsPath, '.sfdx')
-    );
+    return vscode.Uri.file(projectPaths.stateFolder());
```

Comment:
was this already stubbed in the test or is there no test or ?

Reply:
I don't think there was a test for this. Can I write a jest test for this function now?

Reply:
for sure 👍